src/pages/cli/project/troubleshooting.mdx

```diff
@@ -235,6 +235,21 @@ Amplify currently doesn't support automatic data migration. If the data in the o
 If you have previously deployed an application using GraphQL transformer V1 and have decided to migrate to GraphQL transformer V2, go through the [GraphQL transformer migration documentation](/cli/migration/transformer-migration/#changes-that-amplify-cli-will-auto-migrate-for-you) to understand the impact of the migration on the schema directives and limitations prior to updating.
 You can confirm the version of the GraphQL transformer using the `amplify status` command. You can also inspect the `transformerversion` feature flag in `${project-root}/amplify/cli.json`, to confirm.
 
+### Version 11 migration from AWS CDK V1 to V2
+If the Amplify project has been deployed with an Amplify CLI version prior to 11.0.0 that utilizes overrides or custom resources, follow the steps below to migrate:
```

Comment:
Also add some context as to "why" you need to migrate. What's changing? Why does it affect the customer?